### ISSUE TITLE
Issue with the summary_requests.2021_10_01_mobile table

### ISSUE THREAD

**Issue by rviscomi** [HTTPArchive/httparchive.org#477]
According to the [BQ dashboard](https://datastudio.google.com/u/0/reporting/1jh_ScPlCIbSYTf2r2Y6EftqmX9SQy4Gn/page/aLBdB), `summary_requests.2021_10_01_mobile` is missing most of its data. It's expected to have about 650M rows but it only contains 21M. 

![image](https://user-images.githubusercontent.com/1120896/139866193-17714270-9456-4cab-88ae-a4e0b3ce1d7a.png)

**Comment by rviscomi** [comment]
This is causing issues with ismyhostfastyet.com: https://github.com/rviscomi/ismyhostfastyet/issues/46

**Comment by rviscomi** [comment]
I've regenerated the `summary_requests.2021_10_01_mobile` table using the corresponding `requests` HAR data. Here's the query for reference: https://gist.github.com/rviscomi/71f9f5bdd2a0ba8746c30cfa2011a093. As a bonus, I've included a few extra fields that were in the source query used for the Web Almanac (having to do with HTTP version, server push, etc).

There was an issue with the CSV exported from the server; it contained incomplete data. Not sure if that's something we need to worry about for the November crawl but worth keeping an eye on it. This kind of thing has happened in the past and was usually due to some kind of MySQL error, hopefully one that gets blown away by the pre-crawl hard reset.

![image](https://user-images.githubusercontent.com/1120896/141371766-29c76466-391d-4d0e-9d68-a0effa919abc.png)
